blackjax/mcmc/rmh.py

```diff
@@ -74,35 +73,6 @@ def init(position: PyTree, logdensity_fn: Callable) -> RMHState:
     return RMHState(position, logdensity_fn(position))
 
 
-def kernel():
-    """Build a Random Walk Rosenbluth-Metropolis-Hastings kernel with a gaussian
-    proposal distribution.
-
-    Returns
-    -------
-    A kernel that takes a rng_key and a Pytree that contains the current state
-    of the chain and that returns a new state of the chain along with
-    information about the transition.
-
-    """
-
-    def one_step(
-        rng_key: PRNGKey, state: RMHState, logdensity_fn: Callable, sigma: Array
-    ) -> Tuple[RMHState, RMHInfo]:
-
-        move_proposal_generator = normal(sigma)
-
-        def proposal_generator(key_proposal, position):
-            move_proposal = move_proposal_generator(key_proposal, position)
-            new_position = jax.tree_util.tree_map(jnp.add, position, move_proposal)
-            return new_position
-
-        kernel = rmh(logdensity_fn, proposal_generator)
```

Comment:
Then the proposal is accepted using `static_binomial_sampling` of `proposal.py`.